config/WalletConnectorsConfig.tsx

```diff
@@ -0,0 +1,31 @@
+import { InjectedConnector } from 'wagmi/connectors/injected'
+import { WalletConnectConnector } from 'wagmi/connectors/walletConnect'
+import { chain } from 'wagmi'
+
+let kovanURL = "";
+if (process.env.NEXT_PUBLIC_KOVAN_URL) {
+    kovanURL = process.env.NEXT_PUBLIC_KOVAN_URL;
```

Comment:
@ayikfour I'm planning to use hardcoded one. so we don't need the env variable here

 We can limit the access via Alchemy to allow only `risedle.com` can use the rpc url

Reply:
eh tapi kayanya ini di ignore dulu aja ya